src/modules/fancyzones/editor/FancyZonesEditor/CanvasEditorWindow.xaml

```diff
@@ -174,6 +180,11 @@
         </Button>
         <TextBlock Text="{x:Static props:Resources.Name}" Style="{StaticResource textLabel}" />
         <TextBox Text="{Binding Name}" Width="496" Style="{StaticResource textBox}" />
+        <TextBlock Text="Hotkey" Style="{StaticResource textLabel}" />
+        <StackPanel Orientation="Horizontal" Margin="0,8,0,0" HorizontalAlignment="Left">
+        <TextBlock Text="Win + ` + " Style="{StaticResource hotkeyLabel}" />
```

Comment:
Should we be labelling this as "Win + `" or "Alt" as that is the currently linked key?